up42/auth.py

```diff
@@ -109,22 +108,15 @@ def request(
 
         try:
             response: requests.Response = self._request_helper(request_type, url, data)
-            if return_text:
-                try:
-                    response_json = response.json()
-                    # v1 endpoints give response format {"data": ..., "error": ...}
-                    if (
-                        isinstance(response_json, dict)
-                        and response_json.get("error")
-                        and response_json.get("data") is None
-                    ):
-                        raise ValueError(response_json["error"])
-                    # Catalog search, JobTask logs etc. does not have the usual {"data": {}, "error": {}} format.
-                    return response_json
-                except json.JSONDecodeError:  # e.g. JobTask logs are str format.
-                    return response.text
-            else:  # E.g. for DELETE
-                return response
+            try:
```

Comment:
Just a question: 
Why do we need two nested try statements? 
Cannot we have one and multiple `except` depending on the error?

Reply:
The variable used in one of the except blocks is not available at all in the other one. Python does allow that, but it hurts ease of understanding. These methods will be gone soon anyway